packages/server/postgres/queries/getUsersByEmails.ts

```diff
@@ -0,0 +1,22 @@
+import {getUsersByEmailsQuery} from './generated/getUsersByEmailsQuery'
+import getPg from '../getPg'
+import catchAndLog from '../utils/catchAndLog'
+import AuthIdentity from '../../database/types/AuthIdentity'
+import User from '../../database/types/User'
+import {AuthTokenRole} from 'parabol-client/types/constEnums'
+
+export const getUsersByEmails = async (emails: string[]): Promise<User[]> => {
+  return (
+    (await catchAndLog(() => getUsersByEmailsQuery.run({emails}, getPg())))?.map((user) => ({
```

Comment:
let's simplify this to:
```ts
  return getUsersByEmailsQuery.run({emails}, getPg())
```

- the `User` type imported only exists because RethinkDB doesn't have a schema.  `IGetUsersByEmailsQueryResult` will be the single source of truth when RethinkDB goes away :tada: 
- no need for `catchAndLog` because the table exists. at worst it will return an array with `rows.length < emails.length`
-  no need for the `.map` if it returns type `IGetUsersByEmailsQueryResult`. For future reference, if the auto-generated types aren't strict enough we can create a new interface like this:  https://github.com/ParabolInc/parabol/blob/ba49e0e3ba4345857c01c9caa3852b0f51c173a9/packages/server/postgres/queries/getTemplateRefsById.ts#L8-L13. that way we can avoid doing any scrubbing at runtime

Reply:
This leads to type mismatches in a lot of places where we still fetch User from Rethink or PG depending on arguments. I would like to keep it like this for now and change it after I converted the other usages. I updated #5372 to cover this.

Reply:
makes sense. thanks for the update